Tahi-project/tahi
# Bug: Can not add a new reviewer candidate

This is to resolve a regression with the reviewer candidates card during the question restructuring work for metadata snapshots. It got broken in #1803. It doesn't work at all right now on staging.

JIRA issue: https://developer.plos.org/jira/browse/APERTA-5418
#### How to test

First, make sure you have reviewer report nested questions in existence. _This has been run on the review app, so you should only need to do this locally._

```
rake data:migrate:questions-to-nested-questions:reset_all data:migrate:questions-to-nested-questions:migrate_all
```

Then to test:
- You should be able to add reviewer(s) to the reviewer candidates card.
#### Things to note
- This introduces semantic CSS classes for the reviewer. It doesn't remove the _author_ CSS class names because they are used for styling and re-organizing those styles seems out of scope for this since this is to fix a regression in the functionality.
#### Code Reviewer:
- [x] I ran the code locally
#### Product Owner:
- [x] I have verified the expected behavior in the Review environment
